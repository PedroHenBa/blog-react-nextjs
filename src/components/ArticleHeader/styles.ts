import styled, { css } from 'styled-components';
import { Title as HeadingStyles } from '../Heading/styles';

export const Container = styled.div`
  ${({ theme }) => css`
    padding-bottom: ${theme.spacing.xbig};
    margin-bottom: ${theme.spacing.xbig};
    border-bottom: 0.1rem solid ${theme.colors.mediumGray};

    ${HeadingStyles} {
      margin: 0;
      margin-bottom: ${theme.spacing.medium};
    }
  `}
`;

export const Excerpt = styled.p`
  ${({ theme }) => css`
    margin: ${theme.spacing.medium} 0;
    font-size: ${theme.spacing.medium};
  `}
`;

export const Cover = styled.img`
  ${({ theme }) => css`
    max-width: 100%;
    display: block;
    margin-bottom: ${theme.spacing.medium};
  `}
`;
