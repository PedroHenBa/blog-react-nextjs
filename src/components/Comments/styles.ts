import styled, { css } from 'styled-components';

export const Container = styled.div`
  ${({ theme }) => css`
    max-width: ${theme.sizes.content};
    width: 100%;
    margin: ${theme.spacing.big} auto;
    padding: 0 ${theme.spacing.big};
    color: ${theme.colors.darkText};
  `}
`;
