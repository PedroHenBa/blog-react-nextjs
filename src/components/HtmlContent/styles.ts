import styled, { css } from 'styled-components';

export const Container = styled.div`
  ${({ theme }) => css`
    font-size: ${theme.fonts.sizes.medium};

    p {
      margin: ${theme.spacing.big} 0;
    }

    a,
    a:visited,
    a:link {
      color: ${theme.colors.secondary};
      transition: all 200ms ease-in-out;
    }

    a:hover {
      filter: brightness(70%);
    }

    code {
      font-family: monospace;
      color: ${theme.colors.secondary};
      font-size: ${theme.fonts.sizes.small};
      background: ${theme.colors.mediumGray};
      padding: 0.2rem;
      margin: 0.2rem;
    }

    pre {
      background: ${theme.colors.primary};
      padding: ${theme.spacing.medium};
      color: ${theme.colors.white};
      font-family: monospace;
      margin: ${theme.spacing.medium} 0;
      width: 100%;
      overflow-x: auto;
      font-size: ${theme.fonts.sizes.small};
    }

    pre code {
      color: inherit;
      background: inherit;
    }

    img {
      max-width: 100%;
    }

    .image {
      background: ${theme.colors.mediumGray};
      line-height: 0;
      margin: ${theme.spacing.medium} 0;
    }

    .image figcaption {
      padding: ${theme.spacing.small};
      text-align: center;
      font-size: ${theme.fonts.sizes.small};
      line-height: 1.3;
    }

    .image-style-side {
      max-width: 50%;
      float: right;
      margin: ${theme.spacing.medium} 0;
    }

    ul,
    ol {
      margin: ${theme.spacing.medium} ${theme.spacing.xbig};
    }

    hr {
      border: none;
      border: 0.1rem solid ${theme.colors.mediumGray};
    }

    .table {
      width: 100%;
      overflow: hidden;
      overflow-x: auto;
    }

    table {
      border-collapse: collapse;
      border: 1px solid #eee;
      border-bottom: 2px solid #00cccc;
      box-shadow: 0px 0px 20px rgba(0, 0, 0, 0.1), 0px 10px 20px rgba(0, 0, 0, 0.05), 0px 20px 20px rgba(0, 0, 0, 0.05),
        0px 30px 20px rgba(0, 0, 0, 0.05);
      tr {
        &:hover {
          background: #f4f4f4;

          td {
            color: #555;
          }
        }
      }
      th,
      td {
        color: #999;
        border: 1px solid #eee;
        padding: 12px 35px;
        border-collapse: collapse;
      }
      th {
        background: #00cccc;
        color: #fff;
        text-transform: uppercase;
        &.last {
          border-right: none;
        }
      }
    }

    blockquote {
      border-left: 0.5rem solid ${theme.colors.secondary};
      color: ${theme.colors.darkGray};
      filter: brightness(80%);
      padding-left: ${theme.spacing.medium};
      font-style: italic;
      margin: ${theme.spacing.medium};
    }

    @media ${theme.media.lteMedium} {
      font-size: 2rem;

      .image-style-side {
        max-width: 100%;
        float: none;
        margin: 0;
      }
    }
  `}
`;
