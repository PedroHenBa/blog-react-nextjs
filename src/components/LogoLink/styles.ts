import styled from 'styled-components';

export const Container = styled.a`
  display: flex;
  color: inherit;
  text-decoration: none;
  align-items: center;

  > img {
    width: 15rem;
    height: 15rem;
    border-radius: 50%;
  }
`;
