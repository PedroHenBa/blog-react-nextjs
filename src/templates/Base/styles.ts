import styled, { css } from 'styled-components';

export const Container = styled.div``;

export const HeaderContainer = styled.div`
  ${({ theme }) => css`
    margin-top: ${theme.spacing.xhuge};
    padding: 0 ${theme.spacing.big};
  `}
`;

export const ContentContainer = styled.div`
  ${({ theme }) => css`
    max-width: 120rem;
    width: 100%;
    margin: 0 auto;
    margin-bottom: ${theme.spacing.xhuge};
  `}
`;

export const FooterContainer = styled.div`
  max-width: 120rem;
  width: 100%;
  margin: 0 auto;
`;
