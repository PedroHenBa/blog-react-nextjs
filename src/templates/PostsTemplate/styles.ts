import styled, { css } from 'styled-components';

export const Button = styled.button`
  ${({ theme }) => css`
    font-size: ${theme.fonts.sizes.small};
    padding: ${theme.spacing.xsmall} ${theme.spacing.small};
    border-radius: ${theme.spacing.xsmall};
    border: none;
    background: ${theme.colors.primary};
    color: ${theme.colors.white};
    cursor: pointer;

    &:active {
      transform: translateY(-1px);
      box-shadow: 0 5px 10px rgba(0, 0, 0, 0.2);
    }

    &:disabled {
      background: ${theme.colors.darkText};
      color: ${theme.colors.white};
    }
  `}
`;

export const ButtonContainer = styled.div`
  display: flex;
  width: 100%;
  justify-content: center;
`;
