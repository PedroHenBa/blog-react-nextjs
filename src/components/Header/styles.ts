import styled, { css } from 'styled-components';
import { Title as HeadingStyles } from '../Heading/styles';

export const Container = styled.header`
  ${({ theme }) => css`
    display: flex;
    align-items: center;
    justify-content: center;
    padding-bottom: ${theme.spacing.xhuge};
    width: 100%;
    max-width: ${theme.sizes.max};
    color: ${theme.colors.darkText};
    margin: 0 auto;

    ${HeadingStyles} {
      margin: 0 0 calc(${theme.spacing.small} - 1rem);
    }

    @media ${theme.media.lteSmall} {
      display: flex;
      flex-flow: column wrap;
      & > ${HeadingStyles} {
        margin: 0 0 ${theme.spacing.medium} 0;
      }
    }
  `}
`;

export const TextContainer = styled.div`
  ${({ theme }) => css`
    flex-flow: row wrap;
    margin-left: ${theme.spacing.medium};
    max-width: 44rem;

    @media ${theme.media.lteSmall} {
      margin-left: 0;
    }
  `}
`;
