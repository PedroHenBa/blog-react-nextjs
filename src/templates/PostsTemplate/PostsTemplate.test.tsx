import { PostsTemplate } from './index';
import { renderTheme } from '../../styles/render-theme';
import props from './mock';

describe('<PostsTemplate />', () => {
  it('should match snapshot with posts', () => {
    const { container } = renderTheme(<PostsTemplate {...props} />);
    expect(container.firstChild).toMatchSnapshot();
  });

  it('should match snapshot with no posts', () => {
    const { container } = renderTheme(<PostsTemplate {...props} posts={undefined} />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
