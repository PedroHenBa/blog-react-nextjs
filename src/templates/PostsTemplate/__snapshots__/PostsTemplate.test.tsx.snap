// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PostsTemplate /> should match snapshot with no posts 1`] = `
.c9 {
  margin-top: 6.4rem;
  padding: 0 3.2rem;
}

.c13 {
  max-width: 120rem;
  width: 100%;
  margin: 0 auto;
  margin-bottom: 6.4rem;
}

.c19 {
  max-width: 120rem;
  width: 100%;
  margin: 0 auto;
}

.c5 {
  color: #000000;
  line-height: 1.2;
  font-size: 2.4rem;
  text-transform: uppercase;
}

.c12 {
  color: #000000;
  line-height: 1.2;
  font-size: 2.4rem;
  text-transform: none;
}

.c2 {
  z-index: 1;
  position: fixed;
  padding: 3.2rem;
  background: #000000;
  height: 100vh;
  width: 100%;
  max-width: 32rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  top: 0;
  -webkit-flex-flow: column nowrap;
  -ms-flex-flow: column nowrap;
  flex-flow: column nowrap;
  left: -31rem;
  overflow-y: hidden;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c0 {
  z-index: 2;
  position: fixed;
  color: #FFFFFF;
  background: #000000;
  width: 3rem;
  height: 3rem;
  top: 2.4rem;
  color: #FFFFFF;
  left: 0rem;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c7 {
  margin: auto;
  width: 100%;
}

.c3 .c4 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  margin: 0;
  margin-bottom: 4.8rem;
}

.c3 .c4 img {
  border: 0.5rem solid #dc143c;
}

.c6 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  color: inherit;
  -webkit-text-decoration: none;
  text-decoration: none;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c6 > img {
  width: 15rem;
  height: 15rem;
  border-radius: 50%;
}

.c8 {
  display: block;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: #FFFFFF;
  margin-bottom: 1.6rem;
  font-size: 1.8rem;
  border-right: 0.5rem solid #000000;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c8:hover {
  border-right: 0.5rem solid #dc143c;
  color: #dc143c;
}

.c1 {
  display: inline-block;
  vertical-align: middle;
  overflow: hidden;
}

.c10 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  padding-bottom: 6.4rem;
  width: 100%;
  max-width: 96rem;
  color: #333333;
  margin: 0 auto;
}

.c10 .c4 {
  margin: 0 0 calc(1.6rem - 1rem);
}

.c11 {
  -webkit-flex-flow: row wrap;
  -ms-flex-flow: row wrap;
  flex-flow: row wrap;
  margin-left: 2.4rem;
  max-width: 44rem;
}

.c22 {
  font-size: 2.4rem;
}

.c22 p {
  margin: 3.2rem 0;
}

.c22 a,
.c22 a:visited,
.c22 a:link {
  color: #dc143c;
  -webkit-transition: all 200ms ease-in-out;
  transition: all 200ms ease-in-out;
}

.c22 a:hover {
  -webkit-filter: brightness(70%);
  filter: brightness(70%);
}

.c22 code {
  font-family: monospace;
  color: #dc143c;
  font-size: 1.6rem;
  background: #DDDDDD;
  padding: 0.2rem;
  margin: 0.2rem;
}

.c22 pre {
  background: #000000;
  padding: 2.4rem;
  color: #FFFFFF;
  font-family: monospace;
  margin: 2.4rem 0;
  width: 100%;
  overflow-x: auto;
  font-size: 1.6rem;
}

.c22 pre code {
  color: inherit;
  background: inherit;
}

.c22 img {
  max-width: 100%;
}

.c22 .image {
  background: #DDDDDD;
  line-height: 0;
  margin: 2.4rem 0;
}

.c22 .image figcaption {
  padding: 1.6rem;
  text-align: center;
  font-size: 1.6rem;
  line-height: 1.3;
}

.c22 .image-style-side {
  max-width: 50%;
  float: right;
  margin: 2.4rem 0;
}

.c22 ul,
.c22 ol {
  margin: 2.4rem 4.0rem;
}

.c22 hr {
  border: none;
  border: 0.1rem solid #DDDDDD;
}

.c22 .table {
  width: 100%;
  overflow: hidden;
  overflow-x: auto;
}

.c22 table {
  border-collapse: collapse;
  border: 1px solid #eee;
  border-bottom: 2px solid #00cccc;
  box-shadow: 0px 0px 20px rgba(0,0,0,0.1),0px 10px 20px rgba(0,0,0,0.05),0px 20px 20px rgba(0,0,0,0.05),0px 30px 20px rgba(0,0,0,0.05);
}

.c22 table tr:hover {
  background: #f4f4f4;
}

.c22 table tr:hover td {
  color: #555;
}

.c22 table th,
.c22 table td {
  color: #999;
  border: 1px solid #eee;
  padding: 12px 35px;
  border-collapse: collapse;
}

.c22 table th {
  background: #00cccc;
  color: #fff;
  text-transform: uppercase;
}

.c22 table th.last {
  border-right: none;
}

.c22 blockquote {
  border-left: 0.5rem solid #dc143c;
  color: #AAAAAA;
  -webkit-filter: brightness(80%);
  filter: brightness(80%);
  padding-left: 2.4rem;
  font-style: italic;
  margin: 2.4rem;
}

.c20 {
  text-align: center;
  border-top: 0.1rem solid #DDDDDD;
}

.c20 a {
  color: inherit;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c20 .c21 {
  font-size: 1.6rem;
}

.c23 {
  position: fixed;
  background: #000000;
  color: #FFFFFF;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  width: 4rem;
  height: 4rem;
  bottom: 2rem;
  right: 2rem;
  z-index: 6;
}

.c14 {
  width: 100%;
}

.c15 {
  padding: 3.2rem;
  text-align: center;
  font-size: 1.6rem;
}

.c16 {
  margin: 0 auto;
  max-width: 120rem;
  display: grid;
  grid-template-columns: repeat(auto-fill,minmax(280px,1fr));
  gap: 3.2rem;
  padding: 2.4rem;
}

.c18 {
  font-size: 1.6rem;
  padding: 0.8rem 1.6rem;
  border-radius: 0.8rem;
  border: none;
  background: #000000;
  color: #FFFFFF;
  cursor: pointer;
}

.c18:active {
  -webkit-transform: translateY(-1px);
  -ms-transform: translateY(-1px);
  transform: translateY(-1px);
  box-shadow: 0 5px 10px rgba(0,0,0,0.2);
}

.c18:disabled {
  background: #333333;
  color: #FFFFFF;
}

.c17 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  width: 100%;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

@media (max-width:768px) {
  .c2 {
    left: -32rem;
  }
}

@media (max-width:360px) {
  .c10 {
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
    -webkit-flex-flow: column wrap;
    -ms-flex-flow: column wrap;
    flex-flow: column wrap;
  }

  .c10 > .c4 {
    margin: 0 0 2.4rem 0;
  }
}

@media (max-width:360px) {
  .c11 {
    margin-left: 0;
  }
}

@media (max-width:768px) {
  .c22 {
    font-size: 2rem;
  }

  .c22 .image-style-side {
    max-width: 100%;
    float: none;
    margin: 0;
  }
}

@media (max-width:360px) {
  .c16 {
    grid-template-columns: 1fr;
    padding: 1.6rem;
  }
}

<div
  class="styles__Container-sc-550pzn-0"
>
  <a
    aria-label="Open or close menu"
    class="c0"
    href="#"
    title="Open or close menu"
  >
    <svg
      aria-hidden="true"
      aria-label="Open menu"
      class="c1"
      fill="currentColor"
      focusable="false"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M0 0h24v24H0V0z"
        fill="none"
      />
      <path
        d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
      />
    </svg>
  </a>
  <div
    aria-hidden="true"
    class="c2"
  >
    <div
      class="c3"
    >
      <h1
        class="c4 c5"
      >
        <a
          class="c6"
          href="/"
          target="_self"
        >
          <img
            alt="Bloguinho"
            src="https://res.cloudinary.com/djve6xgfe/image/upload/v1627833918/na_14eb8ec6d4.jpg"
          />
        </a>
      </h1>
    </div>
    <nav
      class="c7"
    >
      <a
        class="c8"
        href="link1"
        target="_self"
      >
        text1
      </a>
      <a
        class="c8"
        href="link2"
        target="_self"
      >
        link2
      </a>
    </nav>
  </div>
  <div
    class="c9"
  >
    <header
      class="c10"
    >
      <h1
        class="c4 c5"
      >
        <a
          class="c6"
          href="/"
          target="_self"
        >
          <img
            alt="Bloguinho - am tincidunt ipsum et imperdiet tincidunt. Fusce eget elementum mi. Nulla pharetra aliquet turpis sit amet elementum. In placerat enim ut nequ"
            src="https://res.cloudinary.com/djve6xgfe/image/upload/v1627833918/na_14eb8ec6d4.jpg"
          />
        </a>
      </h1>
      <div
        class="c11"
      >
        <h2
          class="c4 c12"
        >
          Bloguinho
        </h2>
        <p>
          am tincidunt ipsum et imperdiet tincidunt. Fusce eget elementum mi. Nulla pharetra aliquet turpis sit amet elementum. In placerat enim ut nequ
        </p>
      </div>
    </header>
  </div>
  <div
    class="c13"
  >
    <div
      class="c14"
    >
      <p
        class="c15"
      >
        Nenhum post encontrado aqui
      </p>
      <div
        class="c16"
      />
    </div>
    <div
      class="c17"
    >
      <button
        class="c18"
      >
        Carregar mais posts
      </button>
    </div>
  </div>
  <div
    class="c19"
  >
    <footer
      class="c20"
    >
      <div
        class="c21 c22"
      >
        <p>
          Feito com carinho, ph
        </p>
      </div>
    </footer>
  </div>
  <a
    aria-label="Go to top"
    class="c23"
    href="#"
    title="Go to top"
  >
    <svg
      aria-hidden="true"
      class="c1"
      fill="currentColor"
      focusable="false"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M0 0h24v24H0V0z"
        fill="none"
      />
      <path
        d="M7.41 15.41L12 10.83l4.59 4.58L18 14l-6-6-6 6 1.41 1.41z"
      />
    </svg>
  </a>
</div>
`;

exports[`<PostsTemplate /> should match snapshot with posts 1`] = `
.c9 {
  margin-top: 6.4rem;
  padding: 0 3.2rem;
}

.c13 {
  max-width: 120rem;
  width: 100%;
  margin: 0 auto;
  margin-bottom: 6.4rem;
}

.c20 {
  max-width: 120rem;
  width: 100%;
  margin: 0 auto;
}

.c5 {
  color: #000000;
  line-height: 1.2;
  font-size: 2.4rem;
  text-transform: uppercase;
}

.c12 {
  color: #000000;
  line-height: 1.2;
  font-size: 2.4rem;
  text-transform: none;
}

.c2 {
  z-index: 1;
  position: fixed;
  padding: 3.2rem;
  background: #000000;
  height: 100vh;
  width: 100%;
  max-width: 32rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  top: 0;
  -webkit-flex-flow: column nowrap;
  -ms-flex-flow: column nowrap;
  flex-flow: column nowrap;
  left: -31rem;
  overflow-y: hidden;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c0 {
  z-index: 2;
  position: fixed;
  color: #FFFFFF;
  background: #000000;
  width: 3rem;
  height: 3rem;
  top: 2.4rem;
  color: #FFFFFF;
  left: 0rem;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c7 {
  margin: auto;
  width: 100%;
}

.c3 .c4 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  margin: 0;
  margin-bottom: 4.8rem;
}

.c3 .c4 img {
  border: 0.5rem solid #dc143c;
}

.c6 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  color: inherit;
  -webkit-text-decoration: none;
  text-decoration: none;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c6 > img {
  width: 15rem;
  height: 15rem;
  border-radius: 50%;
}

.c8 {
  display: block;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: #FFFFFF;
  margin-bottom: 1.6rem;
  font-size: 1.8rem;
  border-right: 0.5rem solid #000000;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c8:hover {
  border-right: 0.5rem solid #dc143c;
  color: #dc143c;
}

.c1 {
  display: inline-block;
  vertical-align: middle;
  overflow: hidden;
}

.c10 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  padding-bottom: 6.4rem;
  width: 100%;
  max-width: 96rem;
  color: #333333;
  margin: 0 auto;
}

.c10 .c4 {
  margin: 0 0 calc(1.6rem - 1rem);
}

.c11 {
  -webkit-flex-flow: row wrap;
  -ms-flex-flow: row wrap;
  flex-flow: row wrap;
  margin-left: 2.4rem;
  max-width: 44rem;
}

.c23 {
  font-size: 2.4rem;
}

.c23 p {
  margin: 3.2rem 0;
}

.c23 a,
.c23 a:visited,
.c23 a:link {
  color: #dc143c;
  -webkit-transition: all 200ms ease-in-out;
  transition: all 200ms ease-in-out;
}

.c23 a:hover {
  -webkit-filter: brightness(70%);
  filter: brightness(70%);
}

.c23 code {
  font-family: monospace;
  color: #dc143c;
  font-size: 1.6rem;
  background: #DDDDDD;
  padding: 0.2rem;
  margin: 0.2rem;
}

.c23 pre {
  background: #000000;
  padding: 2.4rem;
  color: #FFFFFF;
  font-family: monospace;
  margin: 2.4rem 0;
  width: 100%;
  overflow-x: auto;
  font-size: 1.6rem;
}

.c23 pre code {
  color: inherit;
  background: inherit;
}

.c23 img {
  max-width: 100%;
}

.c23 .image {
  background: #DDDDDD;
  line-height: 0;
  margin: 2.4rem 0;
}

.c23 .image figcaption {
  padding: 1.6rem;
  text-align: center;
  font-size: 1.6rem;
  line-height: 1.3;
}

.c23 .image-style-side {
  max-width: 50%;
  float: right;
  margin: 2.4rem 0;
}

.c23 ul,
.c23 ol {
  margin: 2.4rem 4.0rem;
}

.c23 hr {
  border: none;
  border: 0.1rem solid #DDDDDD;
}

.c23 .table {
  width: 100%;
  overflow: hidden;
  overflow-x: auto;
}

.c23 table {
  border-collapse: collapse;
  border: 1px solid #eee;
  border-bottom: 2px solid #00cccc;
  box-shadow: 0px 0px 20px rgba(0,0,0,0.1),0px 10px 20px rgba(0,0,0,0.05),0px 20px 20px rgba(0,0,0,0.05),0px 30px 20px rgba(0,0,0,0.05);
}

.c23 table tr:hover {
  background: #f4f4f4;
}

.c23 table tr:hover td {
  color: #555;
}

.c23 table th,
.c23 table td {
  color: #999;
  border: 1px solid #eee;
  padding: 12px 35px;
  border-collapse: collapse;
}

.c23 table th {
  background: #00cccc;
  color: #fff;
  text-transform: uppercase;
}

.c23 table th.last {
  border-right: none;
}

.c23 blockquote {
  border-left: 0.5rem solid #dc143c;
  color: #AAAAAA;
  -webkit-filter: brightness(80%);
  filter: brightness(80%);
  padding-left: 2.4rem;
  font-style: italic;
  margin: 2.4rem;
}

.c21 {
  text-align: center;
  border-top: 0.1rem solid #DDDDDD;
}

.c21 a {
  color: inherit;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c21 .c22 {
  font-size: 1.6rem;
}

.c24 {
  position: fixed;
  background: #000000;
  color: #FFFFFF;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  width: 4rem;
  height: 4rem;
  bottom: 2rem;
  right: 2rem;
  z-index: 6;
}

.c14 {
  width: 100%;
}

.c15 {
  margin: 0 auto;
  max-width: 120rem;
  display: grid;
  grid-template-columns: repeat(auto-fill,minmax(280px,1fr));
  gap: 3.2rem;
  padding: 2.4rem;
}

.c16 .c4 {
  margin: 0;
  margin-top: calc(1.6rem - 0.6rem);
  margin-bottom: 1.6rem;
}

.c16 a {
  -webkit-text-decoration: none;
  text-decoration: none;
  color: inherit;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c16:hover a {
  color: #dc143c;
  -webkit-filter: brightness(70%);
  filter: brightness(70%);
}

.c16:hover img {
  -webkit-filter: blur(1px) brightness(100%);
  filter: blur(1px) brightness(100%);
}

.c17 {
  max-width: 100%;
  -webkit-transition: opacity 300ms ease-in-out;
  transition: opacity 300ms ease-in-out;
}

.c19 {
  font-size: 1.6rem;
  padding: 0.8rem 1.6rem;
  border-radius: 0.8rem;
  border: none;
  background: #000000;
  color: #FFFFFF;
  cursor: pointer;
}

.c19:active {
  -webkit-transform: translateY(-1px);
  -ms-transform: translateY(-1px);
  transform: translateY(-1px);
  box-shadow: 0 5px 10px rgba(0,0,0,0.2);
}

.c19:disabled {
  background: #333333;
  color: #FFFFFF;
}

.c18 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  width: 100%;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

@media (max-width:768px) {
  .c2 {
    left: -32rem;
  }
}

@media (max-width:360px) {
  .c10 {
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
    -webkit-flex-flow: column wrap;
    -ms-flex-flow: column wrap;
    flex-flow: column wrap;
  }

  .c10 > .c4 {
    margin: 0 0 2.4rem 0;
  }
}

@media (max-width:360px) {
  .c11 {
    margin-left: 0;
  }
}

@media (max-width:768px) {
  .c23 {
    font-size: 2rem;
  }

  .c23 .image-style-side {
    max-width: 100%;
    float: none;
    margin: 0;
  }
}

@media (max-width:360px) {
  .c15 {
    grid-template-columns: 1fr;
    padding: 1.6rem;
  }
}

<div
  class="styles__Container-sc-550pzn-0"
>
  <a
    aria-label="Open or close menu"
    class="c0"
    href="#"
    title="Open or close menu"
  >
    <svg
      aria-hidden="true"
      aria-label="Open menu"
      class="c1"
      fill="currentColor"
      focusable="false"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M0 0h24v24H0V0z"
        fill="none"
      />
      <path
        d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
      />
    </svg>
  </a>
  <div
    aria-hidden="true"
    class="c2"
  >
    <div
      class="c3"
    >
      <h1
        class="c4 c5"
      >
        <a
          class="c6"
          href="/"
          target="_self"
        >
          <img
            alt="Bloguinho"
            src="https://res.cloudinary.com/djve6xgfe/image/upload/v1627833918/na_14eb8ec6d4.jpg"
          />
        </a>
      </h1>
    </div>
    <nav
      class="c7"
    >
      <a
        class="c8"
        href="link1"
        target="_self"
      >
        text1
      </a>
      <a
        class="c8"
        href="link2"
        target="_self"
      >
        link2
      </a>
    </nav>
  </div>
  <div
    class="c9"
  >
    <header
      class="c10"
    >
      <h1
        class="c4 c5"
      >
        <a
          class="c6"
          href="/"
          target="_self"
        >
          <img
            alt="Bloguinho - am tincidunt ipsum et imperdiet tincidunt. Fusce eget elementum mi. Nulla pharetra aliquet turpis sit amet elementum. In placerat enim ut nequ"
            src="https://res.cloudinary.com/djve6xgfe/image/upload/v1627833918/na_14eb8ec6d4.jpg"
          />
        </a>
      </h1>
      <div
        class="c11"
      >
        <h2
          class="c4 c12"
        >
          Bloguinho
        </h2>
        <p>
          am tincidunt ipsum et imperdiet tincidunt. Fusce eget elementum mi. Nulla pharetra aliquet turpis sit amet elementum. In placerat enim ut nequ
        </p>
      </div>
    </header>
  </div>
  <div
    class="c13"
  >
    <div
      class="c14"
    >
      <div
        class="c15"
      >
        <div
          class="c16"
        >
          <a
            href="/post/qualquer-coisa"
          >
            <img
              alt="Qualquer Coisa"
              class="c17"
              src="https://res.cloudinary.com/djve6xgfe/image/upload/v1627833918/na_14eb8ec6d4.jpg"
            />
          </a>
          <h2
            class="c4 c12"
          >
            <a
              href="/post/qualquer-coisa"
            >
              Qualquer Coisa
            </a>
          </h2>
          <p
            class="styles__Excerpt-sc-1seqxjg-2"
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla non pharetra est, a egestas tellus. Quisque lorem ipsum, lobortis quis ex posuere, ornare placerat tortor. Nulla ultrices eu tortor quis tincidunt.
          </p>
        </div>
        <div
          class="c16"
        >
          <a
            href="/post/salve"
          >
            <img
              alt="Salve"
              class="c17"
              src="https://res.cloudinary.com/djve6xgfe/image/upload/v1627833918/na_14eb8ec6d4.jpg"
            />
          </a>
          <h2
            class="c4 c12"
          >
            <a
              href="/post/salve"
            >
              Salve
            </a>
          </h2>
          <p
            class="styles__Excerpt-sc-1seqxjg-2"
          >
            bi tristique senectus et netus et malesuada fames ac turpis egestas. Proin iaculis turpis sit amet diam vulputate, eget dictum arcu lobortis. Nulla ac susci
          </p>
        </div>
      </div>
    </div>
    <div
      class="c18"
    >
      <button
        class="c19"
      >
        Carregar mais posts
      </button>
    </div>
  </div>
  <div
    class="c20"
  >
    <footer
      class="c21"
    >
      <div
        class="c22 c23"
      >
        <p>
          Feito com carinho, ph
        </p>
      </div>
    </footer>
  </div>
  <a
    aria-label="Go to top"
    class="c24"
    href="#"
    title="Go to top"
  >
    <svg
      aria-hidden="true"
      class="c1"
      fill="currentColor"
      focusable="false"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M0 0h24v24H0V0z"
        fill="none"
      />
      <path
        d="M7.41 15.41L12 10.83l4.59 4.58L18 14l-6-6-6 6 1.41 1.41z"
      />
    </svg>
  </a>
</div>
`;
